hcl/writer.go

```diff
@@ -465,9 +465,21 @@ func writeTuple(pbody, body *hclwrite.Body, attr string, value cty.Value) {
 }
 
 // Interpolate replaces the hardcoded resources link
-// with TF interpolation
+// with TF interpolation.
+// I has the following format:
+// * VALUE => ${aws_instance.name.attribute}
+// So then each value matching the VALUE will
+// be replaced with the reference
 func (w *Writer) Interpolate(i map[string]string) {
-	if !w.opts.Interpolate {
+	// If Interpolation is disabled or
+	// a module without specific attributes to
+	// create as variables (so all are variables),
+	// we ignore the Interpolation as this may happend and is invalid
+	// variable "aws_ses_identity_notification_topic_XoqJb_identity" {
+	//   default = aws_ses_domain_mail_from.SSWXE.id
+	// }
```

Comment:
This is invalid because terraform forbids it? I thought it got addressed https://github.com/hashicorp/terraform/issues/18069 - no?

Reply:
This is a reference, not a variable.